pattern-1/is-deployment.yaml

```diff
@@ -17,15 +17,15 @@ kind: Deployment
 metadata:
   name: wso2is-pattern1-deployment
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
This actually violates our standard deployment pattern - 1 defined for which we say kubernetes artifacts are implemented.
We will need to find a solution for this or at least mention how to get the first deployment pattern done on this mode

Reply:
Please refer earlier comment on the same issue.

Reply:
I added the comment to scale up in README.md. Until the issue is fixed we need to have 2 replicas.